setup.py

```diff
@@ -44,6 +44,9 @@ def get_readme():
         'build_py': get_build_py_cmd(),
         'install': get_install_cmd(),
     },
+    install_requires=[
+        'toml;python_version<"3.11"'
```

Comment:
Do I get it right that `toml` is by default part of Python 3.11+?  Perhaps we could add the dependency unconditionally here?  Alternative would be
`install_requires=conditional_requires`,
and set the `conditional_requires` above `if PythonVersion < 3.11` (pseudocode).

Reply:
`tomllib` is part of Python 3.11+. Other code bases used `tomli` as a drop-in replacement, but that doesn't work with Python 2, so I instead used `toml`, which does.